packages/design-system-dashboard-cli/src/write-comp-usage-to-csv.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+const csv = require('csv');
+const codeowners = require('./codeowners.json');
+const today = require('./today');
+
+/**
+ * @typedef {string} Component
+ * @typedef {string} Path
+ * @typedef {string} Owner
+ * @typedef {number} Total
+ * @typedef {[Component, Path, Owner, Total]} CompPathOwner
+ */
+
+function cleanPath(pathToClean) {
+  const cwd = process.cwd();
+  const relDir = path.relative(cwd, pathToClean).replace(/\.\.\//g, '');
+  return relDir;
+}
+
+/**
+ * @description Should "return" a CSV file where each 'line' has "component, app, owner, total in app"
+ * @param {{breakoutType: string, components: Array<{path: string, component: string}>}} param0
+ */
+function writeCompPathsToCSV({ breakoutType, components }) {
+  // Big reducer takes a component name and path and determines which app and owner they belong to
+  /** @type {CompPathOwner[]} */
+  const compInstPathOwnersOutput = components.reduce(
+    (/** @type {CompPathOwner[]} */ acc, { path, component }) => {
+      let app = '';
+      let owner = '';
+      let total = 1;
+      // First, clean the path of the local user's info
+      let compPath = cleanPath(path);
+
+      // Second, remove the reference to the vets-website if present
+      compPath = compPath.replace('vets-website/', '');
```

Comment:
does this also need to account for the content-build repo? I know there are some component usages in there, owned predominantly by the sitewide team.

Reply:
lol ignore the above comment, just saw that content build is handled below :man_facepalming: